addon/webextension/log.js

```diff
@@ -0,0 +1,47 @@
+/* globals buildSettings */
+
+"use strict";
+
+this.log = (function () {
+  let exports = {};
+
+  const levels = ["debug", "info", "warn", "error"];
+  if (! levels.includes(buildSettings.logLevel)) {
+    console.warn("Invalid buildSettings.logLevel:", buildSettings.logLevel);
+  }
+  let shouldLog = {};
+
+  {
+    let startLogging = false;
+    for (let level of levels) {
+      if (buildSettings.logLevel == level) {
```

Comment:
Mozilla style is unclear on this, e.g., [this](https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Coding_Style) recommends `==`.  The code itself is kind of all over the place.

I changed, but probably best to let some future eslint rule manage this.